test/integration/features/basic.feature

```diff
@@ -147,6 +147,12 @@ Feature: Basic
      """
      exposed
      """
+
+  Scenario: User can access ruby-ex route
```

Comment:
"/" is the root of the application, but maybe in some cases you want to access a URL like /welcome or you maybe want to get pass some parameters. It does not always have to be the root context. Is that clearer now?